lib/public/views/Runs/Details/RunDetailsModel.js

```diff
@@ -49,6 +49,9 @@ export class RunDetailsModel extends Observable {
 
         this._tabbedPanelModel = new RunDetailsTabbedPanelModel();
         this._tabbedPanelModel.bubbleTo(this);
+
+        this._detectorsProvider = createDetectorsProviderInstance();
+        this._detectorsProvider.bubbleTo(this);
```

Comment:
Why?
Again, I think that this should anyway not be in this PR

Reply:
https://github.com/AliceO2Group/Bookkeeping/pull/1288#discussion_r1424232887